torch/distributed/fsdp/_exec_order_utils.py

```diff
@@ -85,8 +78,8 @@ def init(
         for handle in traversal_utils._get_fsdp_handles(root_module):
             index = len(self.all_handles)
             self.all_handles.append(handle)
-            self.handle_to_handle_index[handle] = index
-        self.param_to_fqn = _get_param_to_fqns(root_module)
+            handle._handle_index = index
+            self.param_to_fqn = _get_param_to_fqns(root_module)
```

Comment:
Should this be indented?

Reply:
@voznesenskym Can you check this (possibly in a follow-up)? I don't think this should be indented. It does not hurt correctness but seems like something to fix.

Reply:
I can fix this in https://github.com/pytorch/pytorch/pull/105738/.